critiquebrainz/frontend/templates/macros.html

```diff
@@ -51,3 +51,11 @@
     </script>
   {% endif %}
 {% endmacro %}
+
+{% macro review_credit(review) %}
+{{ _(
+  'Event review by %(user)s' if review.entity_type == 'event' else 'Music review by %(user)s' if review.entity_type == 'release_group' else 'Review by %(user)s',
+  user = '<img class="avatar" src="%s&s=16" /> '|safe % review.user.avatar + '<a href="%s">%s</a>'|safe % (url_for('user.reviews', user_id=review.user.id), review.user.display_name),
+  entitytype = _('Event')
+) }}
```

Comment:
Also, can you update this macro a bit so it doesn't take so much space horizontally? It's hard to read and understand. I'd just replace shortcut conditionals with full versions.